kwasm-lemmas.md

```diff
@@ -82,6 +202,9 @@ Conversely, setting an index in a map to a value `VAL` and then retrieving the v
 
 ```k
     rule #set(BMAP, IDX, #get(BMAP, IDX)) => BMAP [smt-lemma]
+
+    rule #get(#set(BMAP, IDX,  VAL), IDX) => VAL  [smt-lemma]
+    rule IDX ==Int IDX' impliesBool #get(#set(BMAP, IDX', VAL), IDX) ==Int #get(BMAP, IDX) => true [smt-lemma]
```

Comment:
This is a mistake. It gets the final implication to pass, but I really don't understand why.